velox/vector/FlatVector-inl.h

```diff
@@ -83,19 +83,19 @@ xsimd::batch<T> FlatVector<T>::loadSIMDValueBufferAt(size_t byteOffset) const {
 
 template <typename T>
 std::unique_ptr<SimpleVector<uint64_t>> FlatVector<T>::hashAll() const {
-  using len_type = decltype(BaseVector::length_);
+  const vector_size_t length = BaseVector::length_;
```

Comment:
this feels unrelated (but probably harmless).

Reply:
Well, it works with tsan_atomic, but `BaseVector::length_` is tsan_atomic.
We don't want to tsan_atomic will be used for `for (tsan_atomic i = 0; ... ; ++i)`